AdblockPlusSafari/AdblockPlusExtras.m

```diff
@@ -329,4 +287,28 @@ - (BOOL)fileManager:(NSFileManager *)fileManager shouldProceedAfterError:(NSErro
 
 #pragma mark - Private
 
+- (void)onApplicationWillEnterForegroundNotification:(NSNotification *)notification
+{
+  if (self.reloading) {
+    return;
+  }
+
+  __weak __typeof(self) wSelf = self;
+  NSDate *lastActivity = wSelf.lastActivity;
```

Comment:
How to test if content blocker is enabled? Well, it is quite tricky. I observed, that content `reloadContentBlockerWithIdentifier` method return immediately without any error regardless loading has failed if content blocker is disabled. So my solution is force content blocker to produce error and observe if it is returned in response.